r/configure

```diff
@@ -185,6 +185,11 @@ else
   fi
 fi
 
+# If on Raspberry Pi, need to manually link against latomic
```

Comment:
can you add a link to the gcc bug or whatever you found that explains this?

Reply:
Done